packages/web-components/src/components/pictogram-item/__stories__/pictogram-item.stories.ts

```diff
@@ -190,6 +196,7 @@ export default {
           src: selectPictogram(select('Pictogram (required)', pictograms, pictograms.Desktop, groupId)),
           'aria-label': textNullable('Aria-label:', 'Pictogram description', groupId),
         },
+        pictogramColor: select('Pictogram color:', pictogramColors, pictogramColors.black, groupId),
```

Comment:
I'm getting a `error TS2339: Property 'black' does not exist on type '{ 'Black (default)': COLOR_OPTIONS; 'Blue 50': COLOR_OPTIONS; }'.`, which might be cause `pictogramColors.black` is not a define object value. See the code suggestion below.

![Screen Shot 2022-04-07 at 12 12 20 PM](https://user-images.githubusercontent.com/1815714/162265920-8a71b8e1-8b8d-4816-8db3-474d69395ac6.png)


```suggestion
        pictogramColor: select('Pictogram color:', pictogramColors, pictogramColors.COLOR_OPTIONS.DEFAULT, groupId),
```

Reply:
Good catch. `pictogramColors.COLOR_OPTIONS.DEFAULT` didn't work, but plain `COLOR_OPTIONS.DEFAULT` does, so I used that.